spec/features/frontend/affirm_javascript_loading_spec.rb

```diff
@@ -0,0 +1,7 @@
+require 'spec_helper'
+
+RSpec.describe "Affirm javascript" do
+  it "is loaded on the head by default", js: true do
```

Comment:
Well, yes and no. Nothing explicit. I did wanted to make sure we have the affirm js loaded correctly. So when this is passing without any poltergeist js errors it works. It's not clear enough though. Might also not be the best way to test those things.